R/dbz_eta.R

```diff
@@ -20,6 +20,9 @@
 #' dbz_to_eta(7, 5) / 11 # C-band
 #' dbz_to_eta(7, 10) / 11 # S-band
 dbz_to_eta <- function(dbz, wavelength, K = 0.93) {
+  assert_that(class(dbz) == "numeric" & all(dbz>0), msg = "`dbz` must be a strictly positive numeric.")
```

Comment:
Perfect to use `assert_that`. In error message I would use 'single quotes' to indicate literal values, so 'dbz', but we likely haven't been consistent.

Reply:
Actually, ignore that. The styleguide says backticks `...`: https://style.tidyverse.org/error-messages.html so lets use that.